sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_settingselector.py

```diff
@@ -15,7 +15,6 @@ class SettingSelector:
     :type label_filter: str
     """
 
-    def __init__(self, key_filter, label_filter="\0"):
-        # type: (str, str) -> None
+    def __init__(self, key_filter: str, label_filter:str="\0"):
```

Comment:
@annatisch Is it good to list "\0" in the signature?

Reply:
`\0` is the null option, i.e. I want all configurations with a label. Where as None would be all configuration and all labels. So if a configuration existed with multiple labels it would get a result for each.

Reply:
I think that might be clearer if we replace this value in the method signature with some kind of descriptive constant name...

Reply:
We can revisit this during the archboard review.